### ISSUE TITLE
Support 0.10.2.X versions

### ISSUE THREAD

**Issue by Ashish123gs** [yahoo/kafka-manager#394]
I believe we can enable the support for versions 0.10.2.X in Kafka Manager
I am using Kafka Manager for version 0.10.2.1 without any issues .

Thoughts ?

**Comment by ilavender** [comment]
Ashish123gs, were you able to configure kafka-manager consumer for SSL with 0.10.2.1 cluster?

**Comment by keithjpaulson** [comment]
I just set up a test 0.10.2.1 cluster, and all my metrics are zero; the 10.0 clusters registered with that kafka manager all show expected metrics.  (I cannot say for sure it is a 10.2 issue, or if I have a config problem at this point)

**Comment by gena01** [comment]
It seems that KIP-88 still needs to be implemented: https://cwiki.apache.org/confluence/display/KAFKA/KIP-88%3A+OffsetFetch+Protocol+Update
This allows for new API that's available in 0.10.2 via: https://github.com/apache/kafka/pull/2074 which would allow direct API querying of info from broker(s).

**Comment by patelh** [comment]
@gena01 You want to get offsets from brokers using admin client instead of reading them off consumer offset topic ?

**Comment by gena01** [comment]
@patelh Yes. It would provide a more reliable way to see and monitor things. Also would reduce the necessity to scan consumer_offsets topic, which is error prone. 

Wasn't sure if I should open a separate issue for this.

**Comment by patelh** [comment]
@gena01 you realize the consumer_offset topic is where the brokers read offset information ?

**Comment by gena01** [comment]
@patelh I am. And I just ran into this exact scenario. I saw the NPE that's also reported as #404. I was also troubleshooting consumer not showing up in Kafka Manager. Which could mean a broken or buggy consumer. 

When I ran ./kafka-consumer-groups.sh  --bootstrap-server localhost:9092 --list i saw a larger list than what's present in KM. The list included all known consumer groups even if they are not subscribing to any of the topics. Also I would trust the kafka brokers to have a better view on things than trying to scrub the back-end they are writing to.
